backend/app/services/inbox/auto_process_handler.py

```
@@ -0,0 +1,337 @@
+                    )
+                    return
+
+                # Parse autoProcess config
+                spec = work_queue.json.get("spec", {})
+                auto_process_data = spec.get("autoProcess")
+                if not auto_process_data:
+                    return
+
+                try:
+                    auto_process = AutoProcessConfig.model_validate(auto_process_data)
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)